docs/docs/tailwind-css.md

```diff
@@ -105,6 +106,41 @@ const Button = tw.button`
 `
 ```
 
+### Option #3: Sass/SCSS
+
+1.  Install the Gatsby Sass plugin [**gatsby-plugin-sass**](https://www.gatsbyjs.org/packages/gatsby-plugin-sass/) and `node-sass`.
+
+```shell
+npm install --save node-sass gatsby-plugin-sass
+```
+
+_The default Sass implementation wil lbe `node-sass`, but can be changed. Alternatively you can switch to the Dart implementation `sass`:_
```

Comment:
I don't think it's necessary to overcomplicate this guide here and also explain the Dart implementation. That's already covered in the README of the SASS plugin. Let's provide a happy path with `node-sass` and leave a note that for the Dart implementation people should look into the README

Reply:
There's a little typo: "wil lbe" > "will be"

Reply:
Agree with @LekoArts - let's remove lines 117-223 as this just distraction